nussl/core/audio_signal.py

```diff
@@ -705,6 +715,28 @@ def log_magnitude_spectrogram_data(self):
             raise AudioSignalException('Cannot calculate log_magnitude_spectrogram_data '
                                        'because self.stft_data is None')
         return 20 * np.log10(np.abs(self.stft_data) + 1e-8)
+    
+    @property
+    def effects_chain(self):
```

Comment:
Please add a note indicating how you CANNOT use these properties, per our offline discussion.